yt_dlp/extractor/streamtape.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import js_to_json, urljoin
+
+# strings are obfuscated by concatenating substrings
+split_string_part = r'(?:%s|%s)' % (r'"(?:[^"\\]|\\.)*"',
+                                    r"'(?:[^'\\]|\\.)*'")
```

Comment:
cant backreferences be used to simply this?

Reply:
I am not good with regex. Could you may be suggest changes?